Ark.Tools.FtpClient.SystemNetFtpClient/SystemNetFtpClientConnection.cs

```diff
@@ -107,5 +112,23 @@ private System.Net.FtpClient.IFtpClient _getClient()
                 UngracefullDisconnection = false,
             };
         }
+
+        private System.Net.FtpClient.IFtpClient _getClientFromUri()
+        {
+            return new System.Net.FtpClient.FtpClient()
```

Comment:
use a common private as before and locals for Host and Port using an if 

if (uri) host = uri.Host;
etc.